lib/foreman_hooks/util.rb

```diff
@@ -8,14 +8,26 @@ def render_hook_type
       when Host::Managed
         'host'
       else
-        self.class.name.downcase
+        self.class.name.demodulize.downcase
```

Comment:
(Ignore thread, new comment.)

I think downcase should be removed and .tableize (from line 30) should move here.  At the moment this doesn't resolve properly for camel case model names:

<pre>
2.0.0-p353 :003 > ForemanTasks::Task::DynflowTask.name.demodulize.downcase.tableize
 => "dynflowtasks" 
2.0.0-p353 :004 > ForemanTasks::Task::DynflowTask.name.demodulize.tableize
 => "dynflow_tasks" 
</pre>

The first line is what's happening right now, the underscore isn't being preserved.